plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java

```diff
@@ -177,7 +180,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         if (offline) {
             getLog().debug("Executing offline CLI scripts");
             try (StandardOutputStream out = StandardOutputStream.parse(stdout, false)) {
-                final int exitCode = offlineCLIExecutor.execute(jbossHome, getCommands(), getLog(), out, systemProperties);
+
+                String[] opts = javaOpts.split("[\\n\\s]+");
```

Comment:
Thanks. With a simple test I see the same results with or without the `\n`. I'll give the actually build a test and see what happens though. Other than that I think this PR is fine and I'll get it merged here soon.